lib/pangea/choreographer/controllers/choreographer.dart

```diff
@@ -66,6 +70,53 @@ class Choreographer {
     clear();
   }
 
+  List<PreviousMessage> prevMessages() {
```

Comment:
@Kelrap Hey move this to the igc_controller please! Minor thing but to keep choreographer trim. Already a lot there.

Reply:
Also, something like "howFarBack" should be passed as a parameter to a function with a default value.